kubernetes/base/ingress.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
```

Comment:
```suggestion
apiVersion: networking.k8s.io/v1beta1
```

Reply:
Unfortunately this one must stay as it was, because we are using kubectl v1.13. That will result in `error: unable to recognize "kubernetes/kustomize/overlays/production": no matches for kind "Ingress" in version "networking.k8s.io/v1beta1"`

https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/#create-an-ingress-resource